src/editor/config/config.ts

```diff
@@ -9,7 +204,7 @@ export default {
   style: '',
 
   // If true, will fetch HTML and CSS from selected container
-  fromElement: 0,
+  fromElement: false,
```

Comment:
Can we have multiple types here? Being a client-side javascript, use 1/0 as a boolean is preferable

Reply:
@artf Can I ask why? The boolean is more explicit here and it's meant to be used in binary cases like this.

Reply:
> @artf Can I ask why? The boolean is more explicit here and it's meant to be used in binary cases like this.

it's shorten

Reply:
Yeah, but it doesn't make much more sense IMO. Having more readable code is always better than having some sort of shorter code.

Reply:
> Can we have multiple types here? Being a client-side javascript, use 1/0 as a boolean is preferable

This makes no sense. I've never seen 1/0 used as a boolean anywhere else, in any language, client-side or server-side, except for C which doesnt actually have the concept of booleans so it's neccesary. It complicates an aspect of the code that has no reason to be complicated, because you have to expect truthy and falsy values, instead of just `true` and `false`. Even if this PR never gets merged you should really stop using 0 and 1 as booleans.